README.md

```diff
@@ -195,6 +195,31 @@ ollama.embed(request)
   - `options` `<Options>`: (Optional) Options to configure the runtime.
 - Returns: `<EmbedResponse>`
 
+### search
+
+```javascript
+ollama.search(request)
+```
+
+- `request` `<Object>`: The search request parameters.
+  - `queries` `<string[]>`: One or more search queries.
+  - `max_results` `<number>`: (Optional) Maximum results per query (default 5, max 10).
+- Returns: `<SearchResponse>`
+
+Note: Web search requires an API key. Set an `Authorization` header with a bearer token on the client. See Custom Headers. See [examples/websearch/websearch-tools.ts](examples/websearch/websearch-tools.ts) for a usage example with tools.
```

Comment:
Web search requires and Ollama API key - add link to ollama.com where they can get one.